unit-test/test_instructions.py

```diff
@@ -33,6 +33,21 @@ def test_1nnn_jump(self):
             jump = instruction & 0x0FFF
             self.assertEqual(self.ii.program_counter, jump)
 
+    def test_6xkk_set_vx_to_kk(self):
+        self.ii.interpret_instruction(0x62F3)
+        self.assertEqual(self.ii.reg_v[0x2], 0xF3)
+
+        self.ii.interpret_instruction(0x6A12)
+        self.assertEqual(self.ii.reg_v[0xA], 0x12)
+
+    def test_7xkk_add_vx_to_kk(self):
```

Comment:
maybe have a test with two large integers to test the behavior of a byte overflow.